Adding tutorial links to documentation (#1927)
ESMValGroup/ESMValTool: doc/sphinx/source/develop/recipe.rst

    script: /path/to/your/my_little_diagnostic.py

i.e. the full path to the personal diagnostic that the user needs to run.

> There is also an episode available in the 

Reviewer: ```suggestion
There is also a lesson available in the 
```
